README.md

```diff
@@ -21,6 +21,11 @@ In future, we expect gRPC-Web to be supported in
 language-specific Web frameworks, such as Python, Java, and Node. See the
 [roadmap](https://github.com/grpc/grpc-web/blob/master/ROADMAP.md) doc.
 
+Some of the advantages of Protocol Buffers over JSON are:
+
+* Less data to communicate same information
+* Native classes generated for both client and server (so structure and types are built-in)
```

Comment:
I would also mention the possibility to rename fields and change compatible types as that's not possible with the common JSON approach of storing field names in the key.
It would obviously also be possible to do renames by using the tag number as a key, but that's not what the standard [JSON mapping](https://developers.google.com/protocol-buffers/docs/proto3#json) does.